jenkins/scripts/files/run_fetch_logs.sh

```diff
@@ -14,6 +14,24 @@ else
 fi
 mkdir -p ${LOGS_DIR}
 
+# Fetch manifests
+declare -a resource=("bmh" \
+"machine" \
+"m3m" \
+"cluster" \
+"m3cluster" \
+"node")
```

Comment:
that would not be run on the target cluster, so you would get the node of the management cluster (kind or minikube)

Reply:
I would also add all CAPI objects, such as KCP, Machine Deployment, Machine set, Kubeadmconfig (inc. all templates).

Reply:
> I would also add all CAPI objects, such as KCP, Machine Deployment, Machine set, Kubeadmconfig (inc. all templates).

Done.

Reply:
> that would not be run on the target cluster, so you would get the node of the management cluster (kind or minikube)

Do you think we exclude that? Or fetch it from the target cluster? I'm not sure if we really need nodes.

Reply:
you can start without it, and we can add it later on.